### ISSUE TITLE
Convert 1 song at a time

### ISSUE THREAD

**Issue by PyroBillie** [fholger/RocksmithToTab#37]
Hi man, I first of want to thank you, I use your tool alot, especially when I make CDLC from text tab or YT covers. And then I can share the .gp files with the original creators ^^
 
I'd like to give you some feedback tho:
Can you add a function where we can convert 1 song at a time? 
(I thought I found this when I saw the ''Export Guitar Pro'' button in EoF, but that just opens the RocksmithToTab and then starts working on the whole collection)
Cause atm the converter does my whole library and that takes a while, when all I want is the new song I just made :P
(what I do now is: move all my cdlc out of the rocksmith map, run converter, then put them back, which seems unneccesarilly much work just to get 1 tab)
 
EDIT: allright I'm now reading the changelog, and noticed you made a 'batch-process' function, I'll check it out, maybe this fullfills my needs ^^
 
EDIT 2: using v1.1 and I can't chose any directory I want, I wanted to see if I can put my CDLC project directory in there, (to just convert 1 .psarc, but that then gives the error, can't find the songs.psarc)

**Comment by fholger** [comment]
The GUI is rather primitive and only handles the use case of converting the available songs of a Rocksmith installation. However, with v1.1 it does have the option of doing incremental conversions, meaning that only those files should be processed that were updated / added since the last run. 

If that doesn't fit your needs, there's always the command line version which can convert single files or arbitrary directories. In Windows, you should be able to just drag'n'drop a single .psarc file (or a directory with psarc files) onto the RocksmithToTab.exe executable, and it should do what you want.

**Comment by PyroBillie** [comment]
Yes, indeed, Firekorn already mention that you can drag and drop .psarc's on the .exe, that's perfectly for me, thanks ^^
(Not sure if this function is mentioned anywhere, so if that isn't already maybe you should, cause it's really usefull) :D Thanks again

**Comment by HeroMagnus** [comment]
I am having an issue with Convert ONLY NEW files. It goes thru my whole library and overwrites the GP5 files. It was working fine, I would download a new CDLC and it would convert that one file. NOW it goes thru my whole library. So all the changes i made to any existing GP5 file were overwritten. I have a workaround for now but it was so easy just doing new files. Thanks

**Comment by HeroMagnus** [comment]
> I am having an issue with Convert ONLY NEW files. It goes thru my whole library and overwrites the GP5 files. It was working fine, I would download a new CDLC and it would convert that one file. NOW it goes thru my whole library. So all the changes i made to any existing GP5 file were overwritten. I have a workaround for now but it was so easy just doing new files. Thanks

SOLVED: rs2tab.timestamp was NOT being written each time i ran the Converter, I went thru my files and deleted duplicates and m.psarc's

**Comment by fholger** [comment]
Great that you solved it, sorry I couldn't be of any help. It's been a while since I've worked on this :)
